drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-backend/pom.xml

```diff
@@ -143,6 +143,17 @@
       <artifactId>poi</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>org.apache.poi</groupId>
+      <artifactId>poi-ooxml</artifactId>
+      <exclusions>
+        <exclusion>
+          <groupId>stax</groupId>
+          <artifactId>stax-api</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
+
```

Comment:
I think this leaks in from my commit that this PR has. I removed it, but the PR just needs to be refreshed with the latest.

Reply:
removed